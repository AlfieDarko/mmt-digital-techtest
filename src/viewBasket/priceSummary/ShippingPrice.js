import React from 'react'
import { Flex, Heading, Text } from '@chakra-ui/core'

const ShippingPrice = ({ shippingPrice }) => {
  return (
    <>
      <Flex flexDirection="inherit" justifyContent="space-between">
        <Heading as="span" size="sm" >
          Shipping:
            </Heading>
        <Text pl={32} as="span">{shippingPrice}</Text>
      </Flex>
    </>
  )
}

ShippingPrice.defaultProps = {
  shippingPrice: 'free'
}
export default ShippingPrice
