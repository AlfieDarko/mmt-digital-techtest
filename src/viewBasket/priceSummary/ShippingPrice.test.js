import ShippingPrice from './ShippingPrice'
import React from "react";
import { render } from '@testing-library/react'

describe('ShippingPrice', () => {
  it('Should match snapshots', () => {
    const { asFragment } = render(<ShippingPrice price={100} />);
    expect(asFragment()).toMatchSnapshot();
  });
});