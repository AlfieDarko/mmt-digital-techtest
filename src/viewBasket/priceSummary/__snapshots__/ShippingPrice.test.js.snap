// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingPrice Should match snapshots 1`] = `
<DocumentFragment>
  <div
    class="css-13rlfrq"
  >
    <span
      class="css-mgpenw"
    >
      Shipping:
    </span>
    <span
      class="css-o3nqu6"
    >
      free
    </span>
  </div>
</DocumentFragment>
`;
